### PR TITLE
Enable box health filters on Dashboard

### PR DESCRIPTION
**Pull request by CDimonaco**
Co-authored-by: Rubén Torrero Marijnissen <rtorreromarijnissen@suse.com>

# Description

Health Filters in dashboard views

health_box_filters

## How was this tested?

Manually.

### CODE REVIEW

**Comment by abravosuse** [comment]
Code apart, it looks good to me.

**Review by jagabomb — APPROVED** [review]
Nicely done, works as expected.

**Review by arbulu89 — COMMENTED** [review]
Without checking the code, please add some test

**Comment by jagabomb** [comment]
@CDimonaco Can we also add the highlighted hover states for the Health Filter buttons? The hover states are visible in the initial prototype link.

File: assets/js/components/HealthSummary/HomeHealthSummary.jsx [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useState, useEffect } from 'react';
 import Table from '@components/Table';
 import HealthIcon from '../Health/HealthIcon';
 import { Link } from 'react-router-dom';
 import { useSelector } from 'react-redux';
 import HealthSummary from '@components/HealthSummary';
+import useQueryStringValues from '../../hooks/useQueryStringValues';
```
Comment (rtorrero) [inline assets/js/components/HealthSummary/HomeHealthSummary.jsx:7]: I just realized we should probably use `@hooks` in the path for the import here.

**Comment by CDimonaco** [comment]
Yesterday was a little bit hard we skipped tests, but test and hover state are incoming, stay tuned 👀

**Comment by CDimonaco** [comment]
trento_boxes_on_hover

@jagabomb  There are the on hover animation when the boxes are selected

**Comment by CDimonaco** [comment]
Tests added with jest. I also removed old e2e tests for the home dashboard which are now covered in the jest testing files

**Comment by fabriziosestito** [comment]
> Tests added with jest. I also removed old e2e tests for the home dashboard which are now covered in the jest testing files

loving this mega thumbs up

File: assets/js/components/HealthSummary/HomeHealthSummary.test.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import { renderWithRouter, withState } from '../../lib/test-utils';
+import {
+  setHealthSummary,
+  stopHealthSummaryLoading,
+} from '../../state/healthSummary';
+import { act } from 'react-dom/test-utils';
+
+const homeHealthSummaryPayload = [
+  {
+    clusterId: '7965f822-0254-5858-abca-f6e8b4c27714',
```
Comment (arbulu89) [inline assets/js/components/HealthSummary/HomeHealthSummary.test.jsx:17]: Can we use the new factory libraries to create this test data?

**Review by arbulu89 — APPROVED** [review]
The code looks great!

**Review by dottorblaster — APPROVED** [review]
LGTM

File: assets/js/components/HealthSummary/HomeHealthSummary.test.jsx (hunk 2 above)
Comment (CDimonaco) [reply assets/js/components/HealthSummary/HomeHealthSummary.test.jsx]: you're right! done right know!

**Pull request merged**